astronomer/providers/snowflake/operators/snowflake.py

```diff
@@ -71,9 +54,6 @@ def _check_queries_finish(conn: SnowflakeConnection, query_ids: list[str]) -> bo
                 QueryStatus.FAILED_WITH_INCIDENT,
             ):
                 return False
-            else:
-                raise ValueError(f"Unexpected QueryStatus: {str(status)}")
```

Comment:
May I know why should we remove these lines?

Reply:
I'm expecting the query state will be either fail, succeed or running in case of fail/error `get_query_status_throw_if_error` will throw an error and the task will fail and succeed and running case we are handling. About this block, I'm not sure if execution reaches here

Reply:
I think the idea of having this block is to avoid silently passing API changes (e.g., what happen if we change the name or add a new status)

Reply:
I think the snowflake connector provides `is_still_running` method and we should use this method to avoid issues like a new status introduced, wdyt?

Reply:
Sounds good